save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt

```diff
@@ -158,5 +168,38 @@ val cardUser: FC<IndexViewProps> = FC { props ->
                     }
             }
         }
+
+        div {
```

Comment:
I'd also wrap the title and organisations with a `div` and set paddings in high-level `div`

Reply:
Fixed